packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.types.ts

```diff
@@ -40,7 +40,7 @@ export interface IBreadcrumbProps extends React.HTMLAttributes<HTMLElement> {
   items: IBreadcrumbItem[];
 
   /**
-   * Optional root classname for the root breadcrumb element.
+   * Optional class for the root breadcrumb element.
```

Comment:
Most of the changes here are just comment updates. See the next PR comment for the one actual change.